config/config/wc-config.yaml

```diff
@@ -4,99 +4,187 @@ global:
   clusterName: ${CK8S_ENVIRONMENT_NAME}-wc
   baseDomain: "set-me"
   opsDomain: "set-me"
+
+  ## Default cert-manager issuer to use in ingresses
   issuer: letsencrypt-staging
+
+  ## Verify ingress certificates
+  ##
   verifyTls: true
   storageClass: "set-me"
+
+  ## IP of the cluster DNS in kubernetes
+  ##
   clusterDns: "10.43.0.10"
 
+## Object storage configuration for backups.
+##
 objectStorage:
-  # Options are "s3", "gcs", or "none"
-  # If "none", remember to disable backups (velero)
+  ## Options are "s3", "gcs", or "none"
+  ## If "none", remember to disable backups (velero)
   type: "s3"
   # gcs:
   #   project: "set-me"
   # s3:
   #   region: "set-me"
   #   regionAddress: "set-me"
   #   regionEndpoint: "set-me"
+
+  ##
+  ## TODO: remove all but for 'velero'.
+  ##
   buckets:
     harbor: "${CK8S_ENVIRONMENT_NAME}-harbor"
     velero: "${CK8S_ENVIRONMENT_NAME}-velero"
     elasticsearch: "${CK8S_ENVIRONMENT_NAME}-es-backup"
     influxDB: "${CK8S_ENVIRONMENT_NAME}-influxdb"
     scFluentd: "${CK8S_ENVIRONMENT_NAME}-sc-logs"
 
+## User configuration.
 user:
+  ## List of user namespaces to create.
+  ##
   namespaces:
     - demo
+
+  ## List of users to create RBAC rules for.
+  ##
   adminUsers:
     - admin@example.com
+
+  ## Alertmanager configuration.
+  ##
   alertmanager:
     enabled: false
     namespace: monitoring
     ingress:
       enabled: false
 
+
+## Falco configuration.
 falco:
   enabled: true
+
+  ## Define resources requests and limits for single Pods.
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ##
   resources:
     limits:
       cpu: 200m
       memory: 1024Mi
     requests:
       cpu: 100m
       memory: 512Mi
+
+  ## Tolerations for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
   tolerations:
     - key: "node-role.kubernetes.io/master"
       effect: "NoSchedule"
+
+  ## Affinity for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+  ##
   affinity: {}
+
+  ## Node labels for pod assignment
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
   nodeSelector: {}
+
+  ## Falco alert configuration.
+  ##
   alerts:
     enabled: true
-    # supported: alertmanager|slack
+    ## supported: alertmanager|slack
     type: alertmanager
     priority: notice
     hostPort: "http://kube-prometheus-stack-alertmanager.monitoring:9093"
-    # if type=slack falco.alerts.slackWebhook must be set in the secrets yaml file
+    ## if type=slack falco.alerts.slackWebhook must be set in the secrets yaml file
 
+## Prometheus configuration
+##
 prometheus:
   remoteWrite:
-    # Same as 'influxDB.users.wcWriter'.
+    ## Same as 'influxDB.users.wcWriter'.
```

Comment:
This comment is confusing to me. Does it mean that it should be the same as specified for influxdb in sc-config.yaml?
I think it would make more sense to explain what it is: the user used when connecting to the remote write endpoint, by default influxdb in SC.